frontend/app/views/spree/shared/_filters.html.erb

```diff
@@ -1,4 +1,4 @@
-<% filters = @taxon ? @taxon.applicable_filters : [Spree::Core::ProductFilters.all_taxons] %>
+<% filters = @taxon.applicable_filters %>
```

Comment:
Maybe this partial is or was rendered in some view that doesn't have `@taxon`?

Reply:
not anywhere in solidus_frontend, the only time its rendered I can find is here:

https://github.com/solidusio/solidus/blob/master/frontend/app/views/spree/products/index.html.erb#L1-L9

And in that case only if `@taxon` exists.

I personally have a vendetta against these methods because I only barely understand what is happening and don't think they're used or valuable.

Reply:
Oh, yes, seems like it's safe to remove them. I agree they are pretty useless, thanks!